### DISCUSSION TITLE
Mixed solar system

### DISCUSSION THREAD

**Discussion by GPaso** [endless-sky/endless-sky#9414] [Ideas]
I was wandering if it is possible and can have some purpose to have mixed solar systems. For example having a planet with Humans living on it and Quargs living on a moon in the same system. Or a Human colony in a Hay or Remnant solar system?

**Comment by Saugia** [comment]
Technically, this already exists. Tarazed is a good example of this - Wayfarer and Arachne Station are both Republic (Human), and the moon Echo is Quarg. It all depends on the story and if it makes sense to have such a setup.

**Comment by GPaso** [comment]
Oh thanks, I didn't notice!
